Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu
+RUN echo 'Building new SuperDuper Docker image...'
+COPY test/testdata/ca-cert.pem /tmp/
+RUN apt-get install -y software-properties-common && \
```

Comment:
Well, perhaps there should be one before too, but that update is where it is in order to pull the index from the recently added repository.